Source/WebCore/Modules/compression/CompressionStreamEncoder.cpp

```diff
@@ -164,4 +176,82 @@ ExceptionOr<Ref<JSC::ArrayBuffer>> CompressionStreamEncoder::compress(std::span<
 
     return compressedData.releaseNonNull();
 }
+
+
+#if PLATFORM(COCOA)
+ExceptionOr<bool> CompressionStreamEncoder::initializeAppleCompressionFramework() 
+{
+    m_initialized = true;
+    m_usingAppleCompressionFramework = true;
+
+    auto result = compression_stream_init(&m_stream, COMPRESSION_STREAM_ENCODE, COMPRESSION_BROTLI);
+    if (result != COMPRESSION_STATUS_OK)
+        return Exception { ExceptionCode::TypeError, "Initialization Failed."_s };
+
+    return true;
```

Comment:
We never return false so we don't really need 3 bits of data here. Why not just `ExceptionOr<void>`?